docker-compose.override.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  postgres:
+    image: kartoza/postgis:11.0-2.5
+    environment:
+      - POSTGRES_DB=opendatacube
+      - POSTGRES_PASSWORD=opendatacubepassword
+      - POSTGRES_USER=opendatacube
+    ports:
+      - 5432:5432
+    restart: always
```

Comment:
I'd consider building a docker db from a sql dump like I did in datacube-ows instead, so a user doesn't have to re-index everything everytime they want to test it out.

Reply:
This way works fine for testing the UI. Indexing is... another problem. I guess having a pre-indexed DB is nice, but it's not going to be useful for anything.

OWS is different, because you're running tests against that DB.

Reply:
One can test ui without db or docker compose..
To test `dc.load` need **some db**
To test notebooks need **the sandbox db**